raiden/network/proxies/token_network_registry.py

```diff
@@ -104,7 +106,7 @@ def add_token(
         channel_participant_deposit_limit: TokenAmount,
         token_network_deposit_limit: TokenAmount,
         given_block_identifier: BlockIdentifier,
-    ) -> TokenNetworkAddress:
+    ) -> Tuple[TransactionHash, TokenNetworkAddress]:
```

Comment:
I would have put the txhash last in this case. But i guess that's a matter of taste, so no need to change.

Reply:
I preferred the hash being returned first since this is the default for all methods involving transactions - yes, it looks ugly when this method is mostly used for the address - so no strong opinions on that one, but I would leave this as is for now.